addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,311 @@
+---Fields in module 'account'---
+account      / account.account.type     / internal_group (selection)    : NEW selection_keys: ['asset', 'equity', 'expense', 'income', 'liability']
+# NOTHING TO DO
+
+account      / account.analytic.line    / product_uom_id (many2one)     : module is now 'analytic' ('account')
+account      / account.analytic.line    / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+account      / account.bank.statement   / accounting_date (date)        : NEW
+account      / account.bank.statement   / message_last_post (datetime)  : DEL
+account      / account.bank.statement   / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+account      / account.bank.statement.line / account_number (char)         : NEW
+# DONE: post-migration: filled using res.partner.bank acc_number
+
+account      / account.chart.template   / bank_account_code_prefix (char): now required
+account      / account.chart.template   / cash_account_code_prefix (char): now required
+account      / account.chart.template   / transfer_account_code_prefix (char): NEW required: required
+account      / account.chart.template   / transfer_account_id (many2one): DEL relation: account.account.template, required: required
+account      / account.chart.template   / company_id (many2one)         : DEL relation: res.company
+# DONE: post-migration: tried to fill all of them them using the company
+
+account      / account.fiscal.year      / company_id (many2one)         : NEW relation: res.company, required: required, req_default: function
+account      / account.fiscal.year      / date_from (date)              : NEW required: required
+account      / account.fiscal.year      / date_to (date)                : NEW required: required
+account      / account.fiscal.year      / name (char)                   : NEW required: required
+# NOTHING TO DO (new model)
+
+account      / account.group            / parent_left (integer)         : DEL
+account      / account.group            / parent_path (char)            : NEW 
+account      / account.group            / parent_right (integer)        : DEL 
+# DONE: post-migration: call _parent_store_compute() method
+
+account      / account.incoterms        / active (boolean)              : NEW
+account      / account.incoterms        / code (char)                   : NEW required: required
+account      / account.incoterms        / name (char)                   : NEW required: required
+# NOTHING TO DO (rename done in apriori)
```

Comment:
The rename done in apriori is only for comparison purposes unless you add the call to `rename_models` openupgradelib method, which is not done anywhere. We should add that call on `base` pre-migration script, the same as we do with renamed/merged modules. We need to call also to `rename_tables`, so we have to expand the apriori file with this information.

Any way, it seems this doesn't work, or the analysis file is not updated after putting the reference in the renamed models, as the fields are the same than in old model `stock.incoterms`.